src/libstd/process.rs

```diff
@@ -73,6 +73,15 @@ use sys_common::{AsInner, AsInnerMut, FromInner, IntoInner};
 /// spawning process and can itself be constructed using a builder-style
 /// interface.
 ///
+/// There is no implementation of [`Drop`] for child processes,
+/// so if you do not ensure the `Child` has exited then it will continue to
+/// run, even after the `Child` handle to the child process has gone out of
+/// scope.
+///
+/// Calling [`wait`][`wait`] (or other functions that wrap around it) will make
```

Comment:
You don't need to duplicate [`wait`] here, you can drop it, just like [`Drop`] above.